src/main/java/codoc/model/person/Course.java

```diff
@@ -0,0 +1,59 @@
+package codoc.model.person;
+
+import static codoc.commons.util.AppUtil.checkArgument;
+import static java.util.Objects.requireNonNull;
+
+/**
+ * Represents a Person's Course in CoDoc.
+ * Guarantees: immutable; is valid as declared in {@link #isValidCourse(String)}
+ */
+public class Course {
+
+    public static final String MESSAGE_CONSTRAINTS =
+            "Course should only contain alphanumerical characters, and it should not be blank";
+
+    /*
+     * The first character of the Course must not be a whitespace,
+     * otherwise " " (a blank string) becomes a valid input.
+     */
+    public static final String VALIDATION_REGEX = "[a-zA-Z][a-zA-Z ]*";
+
+    public final String course;
```

Comment:
Course as String then using VALIDATION_REGEX to match format.
The first bracket of VALIDATION_REGEX ensures entered course does not begin with a whitespace character.